src/main/java/jenkins/plugins/logstash/configuration/ElasticSearch.java

```diff
@@ -1,202 +1,326 @@
 package jenkins.plugins.logstash.configuration;
 
+import com.cloudbees.plugins.credentials.CredentialsMatchers;
```

Comment:
Please keep source formatting. It is impossible to see what has changed.

Reply:
@mwinter69 really sorry about that! I've fixed this now - the diff should make much more sense.